src/PerfView/GcStats.cs

```diff
@@ -542,43 +543,31 @@ public static void ToXmlAttribs(TextWriter writer, TraceProcess stats, TraceLoad
 
                         if (mt != null)
                         {
-                            writer.Write(" MarkStack =\"{0:n3}", mt.MarkTimes[(int)MarkRootType.MarkStack]);
-                            if (mt.MarkPromoted != null)
+                            void WriteDetailsAboutMarkRootType(MarkRootType type)
```

Comment:
nit - maybe call this something like WriteMarkRootTypeInfo instead of something that reads like a sentence?